scylla/src/network/connection.rs

```diff
@@ -382,6 +418,7 @@ impl Default for HostConnectionConfig {
             tablet_sender: None,
 
             identity: SelfIdentity::default(),
+            pending_request_channel_size: Some(2048),
         }
     }
 }
```

Comment:
The fact that in connection you wrote `.unwrap_or(1024)` (which is a bug) tells me that this approach is too error-prone.

Why use an option here? I don't see anything optional - we need to create the channel, and it needs to have some size. Let's make it just usize.